README.md

```diff
@@ -249,6 +249,13 @@ contentful.createClient({
         Set the environment that the API key has access to.
       </td>
     </tr>
+    <tr>
+      <td><code>additionalTokens</code></td>
+      <td><code>{}</code></td>
+      <td>
+        Additional access tokens, used for resolving links to other spaces. The expected format is <code>{ spaceId: accessToken, ... }</code>
```

Comment:
Also here in the readme maybe make it clear that each spaceId should have their respective access token